<?xml version="1.0" encoding="UTF-8"?>
<sequence name="SapBridgeServiceJmsFaultHandler" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
  <log category="DEBUG" level="custom">
    <property name="SapBridgeServiceJmsFaultHandler" value="Sequence called"/>
    <property name="text" value="An unexpected error occured"/>
    <property expression="$ctx:ERROR_MESSAGE" name="Error because of exception"/>
    <property expression="get-property('ERROR_MESSAGE')" name="message"/>
    <property expression="get-property('ERROR_CODE')" name="code"/>
    <property expression="get-property('ERROR_DETAIL')" name="detail"/>
    <property expression="$ctx:ERROR_EXCEPTION" name="exception"/>
    <property expression="get-property('Action')" name="Action"/>
  </log>
  <makefault version="soap11">
    <code expression="get-property('ERROR_CODE')" xmlns:soap11Env="http://schemas.xmlsoap.org/soap/envelope/"/>
    <reason expression="$ctx:ERROR_MESSAGE"/>
    <role/>
    <detail expression="get-property('ERROR_DETAIL')"/>
  </makefault>
  <header expression="get-property('ReplyTo')" name="To" scope="default"/>
  <property name="RESPONSE" scope="default" type="STRING" value="true"/>
</sequence>
